server/routes/profile/overview/overviewController.test.ts

```diff
@@ -34,7 +34,7 @@ describe('overviewController', () => {
     strengths: [reading, writing, wordFindingNonActive, arithmetic, focussing, tidiness, alphabetOrdering],
   })
   const alnScreeners = setupAlnScreenersPromise({ latestScreener })
-  const curiousAlnAndLddAssessments = Promise.resolve(Result.fulfilled(aCuriousAlnAndLddAssessmentsDto()))
```

Comment:
This was because I'd forgotten the `await` in the middleware above. I should have smelt a rat at the time when I had to wrap the Result in a promise to get the test working! 🙄